src/main/java/com/easysplit/ess/transactions/domain/sql/DebtsQueries.java

```diff
@@ -46,4 +46,17 @@ private DebtsQueries() {
      */
     public static final String GET_LAST_REVISION = "SELECT " + REVISION_COLUMN + " FROM " + TABLE_NAME
             + " WHERE " + DEBTGUID_COLUMN + " = ? ORDER BY " + REVISION_COLUMN + " DESC LIMIT 1";
-}
+
+    /**
+     * Get the debt by id
+     */
+    public static final String GET_DEBT = "SELECT " + DEBTGUID_COLUMN + ", "
+            + TRANSACTIONGUID_COLUMN + ", "
```

Comment:
We can remove this column from here aswell, did not noticed till now but in the WHERE clause, instead of having + `" WHERE " + DEBTGUID_COLUMN +` we should have `+ " WHERE " + TRANSACTIONGUID_COLUMN +` since we want to get the debt details based on it's transactionGuid not the debtGuid